frontend/src/components/edition/editionRowComponent.tsx

```diff
@@ -0,0 +1,100 @@
+import useTranslation from "next-translate/useTranslation";
+import { Col, Container, Row, Toast, ToastContainer } from "react-bootstrap";
+import Image from "next/image";
+import { useState } from "react";
+import { useSWRConfig } from "swr";
+import apiPaths from "../../properties/apiPaths";
+import { StatusCodes } from "http-status-codes";
+import { capitalize } from "../../utility/stringUtil";
+import timers from "../../properties/timers";
+import { editionDelete, extractIdFromEditionUrl } from "../../api/calls/editionCalls";
+import applicationPaths from "../../properties/applicationPaths";
+import useEdition from "../../hooks/useGlobalEdition";
+import { useEditionApplicationPathTransformer } from "../../hooks/utilHooks";
+import { useRouter } from "next/router";
+
+export function EditionRowComponent(props: any) {
+    const { t } = useTranslation("common");
+    const { mutate } = useSWRConfig();
+    const [show, setShow] = useState<boolean>(false);
+    const [contextEdition, setContextEdition] = useEdition();
+    const edition = props.edition;
+    const transformer = useEditionApplicationPathTransformer();
+
+    const router = useRouter();
+
+    if (!edition) {
+        return null;
+    }
+
+    async function changeGlobalEdition() {
+        const replace = router.replace;
+        const query = router.query as { edition?: string };
+
+        setContextEdition(edition._links.self.href);
+
+        replace({
+            query: { ...query, edition: edition.name },
+        }).catch(console.log);
```

Comment:
I'm not sure but can we use the transformer here? (So we don't have a dependency on the format of the edition in the url outside of the Path injector)